MiguelAValdesG/HackerRank/springBootAPIRest/src/main/java/com/bootcamp/springBootAPIRest/service/impl/PatternSyntaxCheckerServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package com.bootcamp.springBootAPIRest.service.impl;
+
+import com.bootcamp.springBootAPIRest.service.PatternSyntaxCheckerService;
+import com.bootcamp.springBootAPIRest.utils.Constants;
+import org.springframework.stereotype.Service;
+
+import org.apache.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.regex.PatternSyntaxException;
+
+@Service
+public class PatternSyntaxCheckerServiceImpl implements PatternSyntaxCheckerService {
+
+  @Override
+  public List<String> patternSyntaxChecker(String[] patterns) {
+    Logger log = Logger.getLogger(Constants.PATTERN_SYNTAX_CHECKER);
+    List<String> response = new ArrayList<>();
```

Comment:
It is recommended that the name of the collections and maps start with singular name and them the name of the interface, example: numberList